### ISSUE TITLE
Wrapper uses readlink -f which is not supported on all versions of macOS

### ISSUE THREAD

**Issue by johnsonjh** [stewartsmith/libeatmydata#28]
The eatmydata wrapper uses readlink -f which is not supported on all versions of macOS - the simplest workaround is to use readlink from coreutils.

I will take a look further at the usage in the script and see if an alternative that does not require any dependencies could be possible unless someone beats me to it.

**Comment by johnsonjh** [comment]
I don't know the exact versions where -f is supported, but the failure noted was from Homebrew's CI on ARM64 macOS 11.

![39DE0B5C-57C4-4AAC-A1FD-24BFDE469AEE](https://user-images.githubusercontent.com/61629094/181364018-6a0f9745-d312-4f06-bce2-076617f6440d.jpeg)

**Comment by johnsonjh** [comment]
Perhaps `realpath -e` would work here.

As with `readlink`, since this is non-POSIX, so I'd need to see if it's more widely supported ...

**Comment by jjwatt** [comment]
FYI, http://mywiki.wooledge.org/BashFAQ/029

tl;dr: You can use `perl`:
```
perl -le 'print readlink "/bin/sh"'
```
or `find` with `printf %l`:
```
$ find /bin/ -type l -printf '%p points to %l\n'
/bin/sh points to bash
/bin/bunzip2 points to bzip2
```
 or write a custom function (that is a little gnarly--see the link for it).
